google/cloud/pubsub/internal/batch_callback.h

```diff
@@ -54,6 +55,10 @@ class BatchCallback {
   virtual void StartScheduler(std::string const& ack_id) = 0;
   virtual void EndScheduler(std::string const& ack_id) = 0;
 
+  virtual Span StartModackSpan(
```

Comment:
`*Span` really doesn't belong in the interface name, nor as the return type. It is an implementation detail of the tracing layer.

Reply:
It seems strange to have a `StartModackSpan` and a `ModackStart`. I don't have a better name than `StartModacks(...)` and `EndModacks()`. WDYT?

Aside: in general, when naming things is hard, it means the interface is confused. But I know we are at the "get this over with and never look back" stage.

Reply:
One represents an event, another represents tarting a span. I can change to StartModackEvent. 

Span does not hold anything in the non-tracing layer. Does it make a difference?